cartridge/failover.lua

```diff
@@ -753,6 +760,25 @@ local function cfg(clusterwide_config)
             end,
         })
         vars.failover_fiber:name('cartridge.stateful-failover')
+    elseif failover_cfg.mode == 'raft' then
+        local ok, err = ApplyConfigError:pcall(raft_failover.cfg)
+        if not ok then
+            return nil, err
+        end
+
+        vars.fencing_enabled = false
```

Comment:
After https://github.com/tarantool/tarantool/commit/ed46feb27087247ab5ee47fcf63f28c3ad842099 there is `election_fencing_enabled` option. I guess it should be supported.